conans/client/migrations.py

```diff
@@ -154,6 +154,8 @@ def _make_migrations(self, old_version):
         if old_version < Version("1.15.0"):
             migrate_registry_file(self.cache, self.out)
 
+        # TODO: Add settings migration
```

Comment:
I don't think we should. The current policy is just to fast forward if not changed. Related with #5038

Reply:
Understood, didn't know about that issue/PR.